Sources/GDSCommon/UI/UIKit/SharedViews/Components/DatePickerViewModel.swift

```diff
@@ -17,15 +17,15 @@ public protocol DatePickerViewModel {
 
 @available(iOS 13.4, *)
 extension DatePickerViewModel {
-    var pickerStyle: UIDatePickerStyle {
+    public var pickerStyle: UIDatePickerStyle {
```

Comment:
Are these changes needed for this PR? can you put a note in the description why your making this change too